migration/azuredeploy-migration.json

```diff
@@ -87,20 +60,6 @@
             },
             "type": "string"
         },
-        "autoscaleVmSku": {
-            "defaultValue": "Standard_DS2_v2",
-            "metadata": {
-                "description": "VM size for autoscaled web VMs"
-            },
-            "type": "string"
-        },
-        "autoscaleVmCountMax": {
-            "defaultValue": 10,
-            "metadata": {
-                "description": "Maximum number of autoscaled web VMs"
-            },
-            "type": "int"
-        },
```

Comment:
Please retain autoscaleVmSku input. Allowing customers to change the target infra VM SKU is in scope of current release.

Reply:
Done, thanks.